sqlx-core/src/postgres/types/mod.rs

```diff
@@ -172,6 +175,8 @@ mod json;
 #[cfg(feature = "ipnetwork")]
 mod ipnetwork;
 
+#[cfg(feature = "chrono")]
+pub use self::chrono::PgTimeTz;
```

Comment:
We've typically not exported the `Pg...` types directly and only used them as an intermediate representation for converting to other types. However, since there is no direct mapping, I _think_ this is okay.